### DISCUSSION TITLE
doc copy not working

### DISCUSSION THREAD

**Discussion by hehehai** [planetscale/discussion#325] [Bug Reports] [answered]
OS: macOs Monterey 12.6 Intel Core i9
Browser: 版本 106.0.5249.119（正式版本） (x86_64)
Page: https://planetscale.com/docs/concepts/planetscale-environment-setup#setup-overview

Reappearance
1. click copy icon
2. show copied
3. paste text -> [object Object]

**Comment by jgreet** [comment]
Thanks for reporting this! Someone from the docs team will take look soon.

**Comment by ayrton** [comment, marked as answer]
We fixed this in November. Thanks again for reporting 🙏
